armi/reactor/blockParameters.py

```diff
@@ -33,8 +33,8 @@ def getBlockParameterDefinitions():
             "orientation",
             units=units.DEGREES,
             description=(
-                "Triple representing rotations counterclockwise around each spatial axis. For example, "
-                "a hex assembly rotated by 1/6th has orientation (0,0,60.0)"
+                "Triple representing rotations counterclockwise around each spatial axis. For "
```

Comment:
same as below....seems like some line wraps get 100 chars applied and some get 120